.travis.yml

```diff
@@ -91,160 +53,150 @@ matrix:
     fast_finish: true
 
     include:
-        # Try MacOS X.
+
+      # Try MacOS X.
         - os: osx
           stage: Cron tests
-          env: SETUP_CMD='test --remote-data=astropy'
-               CONDA_DEPENDENCIES="$CONDA_ALL_DEPENDENCIES clang"
-               PIP_DEPENDENCIES="scikit-image jplephem $ASDF_PIP_DEP"
+          env: PYTHON_VERSION=3.7
+               TOXENV="test-alldeps"
+               TOXPOSARGS="--remote-data=astropy"
```

Comment:
`POSARGS` is odd, here. Really these are extra arguments passed on to the actual code doing the work inside the tox environment. Maybe `TOX_EXTRA_ARGS` or (better) `PYTEST_OPTIONS` (since that seems to be all it is used for).

Reply:
In general I'd be in favor of making this clearer, but since we might one day allow additional arguments to be passed to non-test commands, I don't think ``PYTEST_OPTIONS`` is ideal. In addition once we switch to using https://github.com/astropy/ci-helpers/pull/429 we'll need to use a standard name. The term ``posargs`` has a specific meaning when using tox, but I could see using ``TOX_EXTRA_ARGS`` for readability.

Reply:
If it is a standard environment variable for `tox`, then I'm fine with keeping it. Maybe have a comment on top where those are defined as empty as the default or so?

Reply:
I've added a comment to explain this